Logging/LoggingUtility.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Diagnostics;
+using System.Reflection;
+using log4net;
+using Microsoft.AspNetCore.Http;
+using Newtonsoft.Json;
+
+namespace Logging
+{
+    public class LoggingUtility
+    {
+        private static readonly ILog Logger = LogManager.GetLogger(MethodBase.GetCurrentMethod().DeclaringType);
+
+        private static void LogWithLevel(LogLevels level, string message)
```

Comment:
So the developer has to call `LoggingUtility.LogWithLevel`? 

Does not seem like a clean interface, Why not: `Logger.Info(..)`, `Logger.Trace(..)`, etc

Reply:
The idea was to replicate `req.log.{level}` from fastify, where 
 `req.log.info({ statusCode: 200 }, 'Notify sent!!!!')`
would produce the following output:
`{"level":30,"time":1601367510,"pid":7956,"hostname":"x","reqId":1,"statusCode":200,"msg":"Notify sent!!!!","v":1}` (from Node.js-PrePost-Microservice-Example)

The developer could also instantiate an ILog logger and then use the classic interface:

```
public class HelloWorldController : CpController
{
    private readonly ILog _logger = LogManager.GetLogger(typeof(HelloWorldController));
    [HttpPost]
    [Route("hello")]
    public string LoggingTest([FromBody] Hello hello)
    {
        // Fastify-like log message
        LoggingUtility.LogMessage(HttpContext.Request, LogLevels.Trace, hello, "testMessage");
        // Classic log4net log
        _logger.Trace("testTrace");
    }
}
```